crates/cli-support/src/js/mod.rs

```diff
@@ -426,12 +426,18 @@ impl<'a> Context<'a> {
                         {globals}
                         function init(wasm_path) {{
                             return fetch(wasm_path)
-                                .then(response => response.arrayBuffer())
-                                .then(buffer => WebAssembly.instantiate(buffer, {{ './{module}': __exports }}))
-                                .then(({{instance}}) => {{
-                                    wasm = init.wasm = instance.exports;
-                                    return;
-                                }});
+                                .then(response => response.arrayBuffer(), reason => console.log('Error start fetching', wasm_path, ':', reason))
+                                .then(
+                                    buffer => WebAssembly.instantiate(buffer, {{ './{module}': __exports }}),
+                                    reason => console.log('Error on streaming', wasm_path, ':', reason)
```

Comment:
I don't think it is really appropriate for libraries to log to stdout freely -- shouldn't the consumer of the API be chaining `.then`s that do error logging and all that?

Reply:
Oh, I see! I just don't know how to use a Promise before. This is the first time I see it! OK, I will close this now!